### PR TITLE
[feature-1] First feature was implemented

### PR DESCRIPTION
**Pull request by feathrex**
Following changes where made: 

- new-feature.txt file was added
- feature-1 folder was created

### CODE REVIEW

File: feature1/one-more-file.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch.
```
Comment (feathrex67) [inline feature1/one-more-file.txt:1]: It is a test for multi-user access for repo

Comment (feathrex) [reply feature1/one-more-file.txt:1]: Thanks

Comment (feathrex67) [reply feature1/one-more-file.txt:1]: Another test

Comment (feathrex) [reply feature1/one-more-file.txt:1]: More testing is needed.

**Review by feathrex67 — APPROVED** [review]
Final comment. Approved.

**Pull request merged**